### PR TITLE
Fix handling of socket connection refusal.

### PR DESCRIPTION
**Pull request by rhatdan**
Currently if the socket was never started you get an error about
the service being started.  But if the service was started and later
stopped, you get a useless error.

This change causes the error to always be the same for connection refused.

The error message was also repeating the address twice which looked bad.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@jwhonce @baude PTAL
BTW pypodman does not work in release because we did not install python3-psutil as a requires in Fedora or RHEL spec.

**Comment by baude** [comment]
LGTM

File: contrib/python/podman/podman/client.py [hunk 1]
```diff
@@ -175,12 +175,12 @@ def __init__(self,
             if not System(self._client).ping():
                 raise ConnectionRefusedError(
                     errno.ECONNREFUSED,
-                    'Failed varlink connection "{}"'.format(address), address)
+                    ('Failed varlink connection "{}"').format(address))
```
Comment (jwhonce) [inline contrib/python/podman/podman/client.py]: Why this change? The second address populates the data field in the Error

File: contrib/python/podman/podman/client.py [hunk 2]
```diff
@@ -175,12 +175,12 @@ def __init__(self,
...
         except FileNotFoundError:
             raise ConnectionError(
                 errno.ECONNREFUSED,
                 ('Failed varlink connection "{}".'
-                 ' Is podman service running?').format(address), address)
+                 ' Is podman socket or service running?').format(address))
```
Comment (jwhonce) [inline contrib/python/podman/podman/client.py]: Additional information in Error message LGTM, again why remove data field in Error?

**Comment by jwhonce** [comment]
LGTM

@rhatdan I'll put together a PR for the spec file update.

**Comment by baude** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@rh-atomic-bot retry

**Comment by rhatdan** [comment]
@jwhonce The reason the second field was removed, was that it was printing twice.  The first field was enough to get it printed.
